server/requirements.txt

```diff
@@ -13,7 +13,7 @@ configparser>=3.5.0,<3.6; python_version == '2.7'
 coverage==4.5.2
 Django>=1.11.20,<3
 django-chartjs==1.3
-djangorestframework>=3.9.4,<3.10
```

Comment:
Have you checked why we added this requirement in the first place? I believe there was some sort of incompatibility with our code?

Reply:
The functional changes in 3.10 and 3.11 were around schema support, which we don't use at all. 3.10 dropped support for Python 2.7, which is why I think we specified `<3.10`.